nvm.sh

```diff
@@ -146,7 +146,12 @@ nvm_has_system_iojs() {
 }
 
 nvm_is_version_installed() {
-  [ -n "${1-}" ] && [ -x "$(nvm_version_path "$1" 2>/dev/null)"/bin/node ]
+  local NVM_OS
+  NVM_OS="$(nvm_get_os)"
+  local NODE
+  NODE='node'
+  [ "_${NVM_OS}" = "_win" ] && NODE='node.exe'
```

Comment:
will this break WSL users?

Reply:
On WSL you should Linux as OS, at least that is what it does for me, so it should not break anything.
By the way I have tried on WSL, and it does not work due to some \r\n end of line on script from npm installed on Windows. I guess it would work better with an npm install directly under WSL, I havent tried yet,

Reply:
Yes, on WSL, node/npm must only be installed on WSL or else you get those kinds of problems.